### ISSUE TITLE
laravel 53 publish error

### ISSUE THREAD

**Issue by illuminate3** [shvetsgroup/laravel-email-database-log#2]
This is the error I get on the command line.

```
 [ErrorException]
  Declaration of ShvetsGroup\LaravelEmailDatabaseLog\LaravelEmailDatabaseLogServiceProvider::boot(Illuminate\Contracts\E
  vents\Dispatcher $events) should be compatible with Illuminate\Foundation\Support\Providers\EventServiceProvider::boot
  ()
```

Are you planning to update your package to 5.3?

Thanks!

**Comment by d-lamers** [comment]
I have created a pull request to fix this issue with Laravel 5.3

https://github.com/shvetsgroup/laravel-email-database-log/pull/3

**Comment by neochief** [comment]
Fixed in 1.0.3.

**Comment by codetroll** [comment]
I get the same error now in L5.2 so I guess the fix to make it work with 5.3 broke 5.2?
